static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -75,7 +75,11 @@ define([
                 },
                 shouldSendCallback: function (data) {
                     var isDev = config.page.isDev;
-                    if (isDev) {
+
+                    var isIgnored = typeof(data.tags.action) !== 'undefined' &&
+                        config.page.ignoredSentryActions.indexOf(data.tags.action) != -1;
```

Comment:
`!==` on the second comparison here?

Also is undefined specifically a special case here?  If not: 

    var isIgnored = data.tags.action && ...